adaptive_navigation/test/adaptivenavigationscaffold_test.dart

```diff
@@ -1,7 +1,5 @@
 import 'package:flutter_test/flutter_test.dart';
 
-import 'package:adaptive_navigation_scaffold/scaffold.dart';
```

Comment:
`import 'package:adaptive_navigation/scaffold.dart';` is correct, but it is not used now, so I deleted it.